composer.json

```diff
@@ -23,6 +23,7 @@
   },
   "require": {
     "drupal/commerce": "~2.0",
+    "drupal/commerce_currency_resolver": "~1.0",
```

Comment:
**CHANGE REQUEST**

That should be added as a suggested dependency as it's not mandatory if you don't enable the currency submodule.

Reply:
@krystalcode How do we add as suggested dependency? I removed it from composer.json. Not sure on how to add as suggested dependency

Reply:
@safalliajoseph https://getcomposer.org/doc/04-schema.md#suggest

Reply:
Added as suggestion